docs/sql-reference/sql-statements/data-definition/DROP TABLE.md

```diff
@@ -1,21 +1,21 @@
 # DROP TABLE
 
-## description
+## Description
 
 This statement is used to delete a table.
 
-Syntax:
+## Syntax
 
 ```sql
 DROP TABLE [IF EXISTS] [db_name.]table_name [FORCE];
 ```
 
 Note:
 
-1. After executing DROP TABLE for a while, you can use RECOVER statement to restore the dropped table. See RECOVER statement for more detail.
-2. If DROP Table FORCE is executed, the table will be deleted directly and cannot be recovered without checking whether there are unfinished activities in the database.  Generally this operation is not recommended.
+- If it has not been more than a day since a table was deleted by using the DROP TABLE statement, you can use the [RECOVER](../data-definition/RECOVER.md) statement to restore the table.
```

Comment:
You should pay attention on this RECOVER page:
"It can only recover meta-information deleted some time ago", can it only recover meta-information? can't it recover the data? you can check it from RDs

Reply:
@jaogoy 
Confirmed the following info with simo:
For example, after executing the DROP TABLE statement to delete a table,  SR first deletes the metadata of this table. Then if it has been more than a day since a table was deleted, SR will delete the data in this table. 

That's why "If it has not been more than a day since a table was deleted by using the DROP TABLE statement, you can use the [RECOVER](../data-definition/RECOVER.md) statement to restore the table." 

The description in the RECOVER statement is incorrect and need to be revised.

Reply:
OK.
we should say something in the user's shoes: DROP will not just delete the metadata, it will delete the data too, all of them are moved to trash bin.